Monika After Story/game/script-topics.rpy

```diff
@@ -546,6 +546,32 @@ label monika_selfesteem:
             m 3hub "I'll always root for you, [player]!"
     return "derandom"
 
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="songwriting", 
+            category=["music"],
+            prompt="Songwriting", 
+            random=True, 
+            pool=True
+        )
+    )
+
+label monika_example:
+    m 3eub "Hey [player], have you ever written a song?"
+    m 3ekb "I've tried myself, but it's hard to do with my limited resources here."
+    m 4eub "You know, I used to write some from time to time in my sophmore years. I used to be pretty good."
+    m 3eksdla "But, I guess it's been a while, huh?"
+    m 1eksdla "A-anyway..."
+    m 5eubla "Would you ever consider writing me a song?"
+    m 1eub "I think that'd be a great gift, if you're planning for something!"
+    m 4eubfa "I'm just saying..."
```

Comment:
This line here is what also causing you to fail, mainly just because it's using a sprite which isn't added to the game. That's fine, we'll just add that later.